shell/platform/fuchsia/runtime/dart/utils/root_inspect_node.cc

```diff
@@ -21,4 +21,8 @@ inspect::Node RootInspectNode::CreateRootChild(const std::string& name) {
   return inspector_->inspector()->GetRoot().CreateChild(name);
 }
 
+inspect::Inspector* RootInspectNode::GetInspector() {
```

Comment:
This will likely collide with https://github.com/flutter/engine/pull/26295

Reply:
yup! I'll sync up with naud about this.